Project.toml

```
@@ -11,7 +11,7 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 [compat]
 IterTools = "1"
 Reexport = "1"
-StatsBase = "0.33"
+StatsBase = "0.34"
```

```suggestion
StatsBase = "0.33, 0.34"
```
The change made is compatible with 0.33 as well